mongodb-backend/.gitignore

```diff
@@ -1,2 +1,3 @@
 /node_modules
+package-lock.json
```

Comment:
This should be commited, it's long but it specifies all the versions of each package you're using so that when someone else tries to run the app, they use the exact same packages

Reply:
Removed it because it looked ugly. **Added back**